Header header

# distance between slices.
float32 dist_increment

# vector of slices at the given increment
# note, the first index is usually considered 0 distance, and is empty.
LidarSpaceSlice[] ranges
